examples/fluids/navierstokes.h

```diff
@@ -194,7 +194,7 @@ struct Physics_private {
   EulerTestType            euler_test;
   StabilizationType        stab;
   PetscBool                implicit;
```

Comment:
It isn't pressing or anything, but this PR highlights a few places where we aren't using the `PetscBool verb_description` style for our booleans in fluids. I'm sure we're get them all eventually.

Reply:
Yeah, I've been trying to fix them when I touch them.